misc/config_tools/schema/checks/rdt_support.xsd

```diff
@@ -27,15 +27,15 @@
                   )
                   ">
     <xs:annotation>
-      <xs:documentation>For a vCAT VM, number of clos/vcpu_clos elements must be greater than 1!</xs:documentation>
+      <xs:documentation>For a VM with Virtual Cache Allocation Technology enabled, you must define more then one clos/vcpu_clos element.</xs:documentation>
```

Comment:
We removed clos/vcpu_clos in the UI. Would any clos-related errors occur in the UI? If so, how would a user recover? This comment applies to all messages that mention clos or vcpu_clos below.

Reply:
Alas, I have no idea how the user would recover from this assertion failure since they have no way to change these settings with the current UI.  @junjiemao1 any ideas?

Reply:
That assertion should be dropped as those config items involved no longer exist.

Reply:
OK.  I'll delete the assert